lib/realtime/tenants/connect.ex

```diff
@@ -88,10 +143,28 @@ defmodule Realtime.Tenants.Connect do
 
   ## Private functions
 
-  defp call_external_node(tenant_id) do
+  defp call_external_node(tenant_id, opts) do
     with tenant <- Tenants.Cache.get_tenant_by_external_id(tenant_id),
          {:ok, node} <- Realtime.Nodes.get_node_for_tenant(tenant) do
-      :erpc.call(node, __MODULE__, :connect, [tenant_id], 5000)
+      :erpc.call(node, __MODULE__, :connect, [tenant_id, opts], @erpc_timeout_default)
     end
   end
+
+  defp update_connected_users_bucket(tenant_id, connected_users_bucket) do
+    connected_users_bucket
+    |> Enum.drop(1)
+    |> then(&(&1 ++ [UsersCounter.tenant_users(tenant_id)]))
```

Comment:
```suggestion
    connected_users_bucket
    |> then(&(&1 ++ [UsersCounter.tenant_users(tenant_id)]))
    |> Enum.take(-6)
```

Double check this :D

Reply:
And then here:

```
[UsersCounter.tenant_users(tenant_id) | connected_users_bucket]
|> Enum.take(6)
```